.github/workflows/scheduled_builds.yml

```diff
@@ -1,60 +1,16 @@
-name: "Build Edge on Schedule"
+name: Build Edge on Schedule
 
 on:
   schedule:
-    - cron: "0 0 * * 5"
+    - cron: 0 0 * * 5
```

Comment:
The original intend was to provide up-to-date package versions to users using `:edge`, which includes updated to ClamAV I guess, but because we have a cron job inside the container that runs `freshclam` daily, I think this is not a concern.

The `COPY --link ..` is best suited for another PR :) We may be able to get rid of this workflow if we really want to, but right now, I see no harm. The cache problem might however be a real concern. I you're right and I understood you correctly, we do not get package updates because we will now use a cache, which "skips" package installation. We should solve this somehow, otherwise this workflow really is useless.